ArkhamOverlay/Pages/Main/MainView.xaml

```diff
@@ -96,7 +114,12 @@
                 <Button Margin="0, 10, 0, 0" Width="200" Command="{Binding ShowEncounterDeck}">Encounter Deck</Button>
             </StackPanel>
         </StackPanel>
-
+        <StackPanel Orientation="Horizontal" Margin="0, 10, 0, 0">
+            <TextBlock Text="Track Player Values: " />
+            <CheckBox Margin="10, 1, 0, 0" IsChecked="{Binding Configuration.TrackHealthAndSanity}">TrackHealth/Sanity</CheckBox>
```

Comment:
I think this checkbox can just be named "Health/Sanity"

Reply:
I ended up fixing this later (next pull request)